src/deepsparse/server/config.py

```diff
@@ -49,6 +49,63 @@ class ImageSizesConfig(BaseModel):
     )
 
 
+class SystemLoggingGroup(BaseModel):
+    """
+    Holds the configuration for a single system logging group.
+    """
+
+    enable: bool = Field(
+        default=False, description="Whether to enable the system logging group"
+    )
+
+    target_loggers: Optional[List[str]] = Field(
+        default=None,
+        description="The list of target loggers to log to. "
+        "If None, logs to all the available loggers",
+    )
+
+
+class SystemLoggingConfig(BaseModel):
+    """
+    Holds the configuration for the system logging
+    """
+
+    # Global System Logging
+    enable: Optional[bool] = Field(
+        default=True, description="Whether to enable system logging"
+    )
+    target_loggers: Optional[List[str]] = Field(
+        default=None,
+        description="The list of loggers to log to. If None, "
+        "logs to all the available loggers.Will "
+        "be overriden by the target_loggers specified "
+        "in the individual system logging groups.",
+    )
```

Comment:
What about this?
```suggestion
    # Global System Logging
    system: SystemLoggingGroup = Field(..., default=SystemLoggingGroup(enable=True))
```

Reply:
I think having target loggers at this level is overkill here. we can just delegate the user to the global target loggers and then can filter in the individual group configs. no need IMO for 3 levels of filtering target loggers. clutters config and may lead to confusion on hierarchy.

Reply:
Fair point, I agree.